src/pages/core/guides/loyalty-card-concept-guide.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Loyalty Card Concept Guide
+metaTitle: Loyalty Card Concept Guide | Core Guides
+description: This guide describes how to build a Loyalty Card program on Solana using MPL Core NFT Assets and the MPL Core plugin system.
+---
+
+## Concept Guide: Setting Up Loyalty Cards with Metaplex Core and Plugins
+
+> ⚠️ This is a **concept guide**, not a complete end-to-end tutorial. It is intended for developers with a working understanding of Rust and Solana, particularly using the Anchor framework. While it walks through key architectural decisions and code examples, it assumes familiarity with program structure, CPIs, and deploying Solana smart contracts.
+
+This guide assumes you have some basic knowledge of Solana and Rust using Anchor. It explores one way to implement a loyalty card system using Core NFT Assets on Solana, powered by Metaplex Core. Rather than prescribing a rigid approach, this guide aims to demonstrate a flexible pattern that you can adapt to your own project.
+
+### What is Metaplex Core?
+Metaplex Core is a modern NFT Asset standard on Solana that provides a plugin-based architecture. Unlike the legacy Token Metadata program, Core allows developers to attach modular functionality to NFTs, such as custom data storage, ownership controls, and rule enforcement.
+
+In this example, you'll use three components from Metaplex Core:
+- **AppData Plugin**: To store custom structured data (like loyalty points).
+- **Freeze Delegate Plugin**: To lock NFTs, so users cannot transfer or burn them (soulbound behavior).
+- **Update Delegate Authority (via PDA)**: To give your program control to update child NFTs minted under a specific collection.
+We'll also use **CPI builders** (e.g., `CreateV2CpiBuilder`) to interact with the Metaplex Core program. These builders simplify how you construct and invoke instructions, making code easier to read and maintain.
+
+### Quick Lifecycle Overview
+```
+[User] → requests loyalty card
+    ↓
+[Program] → mints NFT + AppData + FreezeDelegate (soulbound)
+    ↓
+[User] → purchases coffee or redeems points
+    ↓
+[Program] → updates loyalty data in AppData plugin
+```
+
+See the [Metaplex Core documentation](https://developers.metaplex.com/core) for more setup details.
+
+---
+
+## Loyalty System Architecture
+
+This example outlines one potential structure for creating a loyalty card system using Metaplex Core on the Solana blockchain. The loyalty cards are NFTs, each associated with plugins that manage how they behave and store data.
+
+### Why Use Soulbound NFT Assets?
+
+Making loyalty cards soulbound helps ensure that they're tied to a single user and can't be transferred or sold. This can help preserve the integrity of the loyalty program and prevent users from gaming the system by trading or duplicating rewards.
+
+### LoyaltyCardData Structure
```

Comment:
Somehow i feel like this jumps into code too early without first properly describing what the program / page does?